assets/src/components/App.jsx

```diff
@@ -11,44 +12,52 @@ import trackAnalyticsPageView from '../utils/analytics';
 class App extends Component {
   constructor(props) {
     super(props);
+
+    this.updateUser = () => {
+      getLoggedUser().then((user) => {
+        this.setState({ user });
+      });
+    };
+
     this.state = {
       user: null,
+      updateUser: this.updateUser,
```

Comment:
Why do you need to set a function in the state?

Reply:
Both good points! Thanks for the review ;)